dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jakarta-connectionManagement-1.0.feature

```diff
@@ -4,7 +4,7 @@ IBM-Provision-Capability:\
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.transaction-2.0)))", \
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.connectionManagement-1.0))"
 IBM-Install-Policy: when-satisfied
--features=com.ibm.websphere.appserver.jakarta.connector-2.0
+-features=com.ibm.websphere.appserver.jakarta.connector.internal-2.0
 -bundles=com.ibm.ws.jca.cm.jakarta
-kind=noship
```

Comment:
that doesn't say beta!!

Reply:
It says noship because you commented on the old version of the file rather than the new one.  When you look at the new version, it says ga, which is due to Andy's request to switch autofeatures from beta to ga, so this can be lumped in to that discussion.

Reply:
lol that was weird, when i wrote that comment, it said ga!